AzureMonitorAgent/ama_tst/modules/logcollector.py

```diff
@@ -54,9 +56,10 @@ def is_arc_installed():
 
 # Log collecting functions
 
-def collect_logs(output_dirpath):
-    # collect MDSD logs
-    copy_dircontents("/var/opt/microsoft/azuremonitoragent/log", os.path.join(output_dirpath,"mdsd"))
+def collect_logs(output_dirpath, pkg_manager):
```

Comment:
Should not you also change the tool's version number ?

Reply:
Yes apologies, forgot to update the version number, just fixed it!